src/main/java/com/epam/esm/service/TagService.java

```diff
@@ -17,43 +21,29 @@
 @Service
 public class TagService {
 
-    private final GiftCertificateTagRepository giftCertificateTagRepository;
-
+    private final TagRepository tagRepository;
 
     @Autowired
-    public TagService(GiftCertificateTagRepository giftCertificateTagRepository) {
-        this.giftCertificateTagRepository = giftCertificateTagRepository;
+    public TagService(TagRepository tagRepository) {
+        this.tagRepository = tagRepository;
     }
 
-    /**
-     *
-     * @param tagName name of the tag to be saved.
-     * @return
-     * if tag name is not valid, returns bad request
-     * if tag already exists, returns bad request
-     * if tag does not exist, but cannot be saved, return bad request
-     * if it is saved, return CREATED and tag saved
-     */
+
```

Comment:
Why did you remove the javadoc here? You would need to restore it later.

Reply:
Because I'm making changes to the implementation and some fields are changing, what gives me error in javadoc. When I finish everything I would just copy from last project and paste, if there were no changes.